src/components/ValidationMessage/ErrorValidation.tsx

```diff
@@ -0,0 +1,25 @@
+import ErrorIcon from './ErrorIcon'
+
+type Props = {
+  clickCallback: () => void
+}
+
+const ErrorValidation = ({ clickCallback }: Props) => {
```

Comment:
```suggestion
import { FC } from 'react';

interface ErrorValidationProps = {
  onRetry: () => void
}

const ErrorValidation: FC<ErrorValidationProps> = ({ onRetry }) => {
```

Apart from the suggestions made above, I'd also recommend some work on the naming. In my opinion, `clickCallback` doesn't provide much information about the parameter itself. Following clean code guidelines, I would rename it to something more significant, such as `onRetry`.

Typically, parameters that expect a callback function are prefixed by `on`. By renaming the prop to `onRetry` you are providing some context to an external user of this component: the function you pass as a prop, is going to be called whenever there is a retry

`onRetry` is just a suggestion. You can choose any other name, but I do recommend using the `on` prefix with whatever callback